db.js

```diff
@@ -0,0 +1,47 @@
+const uuid = require('uuid/v1');
+const db = [
```

Comment:
It is better to use object since you can access particular user by key by single operations and not to filter a whole array.